src/components/TripItem.jsx

```diff
@@ -19,20 +20,44 @@ import {
   getFormattedDuration,
   getModes,
   formatTripDistance,
-  getStartDate,
-  getMainMode
+  getStartDate
 } from 'src/lib/trips'
 import { computeCO2Trip } from 'src/lib/metrics'
-import { pickModeIcon, modeToColor } from 'src/components/helpers'
+import {
+  pickPurposeIcon,
+  purposeToColor,
+  pickModeIcon
+} from 'src/components/helpers'
 import TripDialogDesktop from 'src/components/Trip/TripDialogDesktop'
 
 const styles = {
   co2: { fontWeight: 700 }
 }
 
+const getIconStyle = (idx, icons) => ({
+  margin: idx !== 0 && idx < icons.length - 1 ? '0 4px' : '0'
+})
+
+const TripItemSecondary = ({ tripModeIcons, duration, distance }) => {
+  return (
```

Comment:
not necessary return (we should add a eslint rule in the linter)

Reply:
A débattre au prochain point, je ne pas pour ajouter ce genre de règle :)